tool/tctl/common/auth_command.go

```diff
@@ -252,8 +255,28 @@ func (a *AuthCommand) ExportAuthorities(ctx context.Context, clt authCommandClie
 		return trace.Wrap(err)
 	}
 
-	fmt.Println(authorities)
+	if l := len(authorities); l > 1 && a.exportOutputPrefix == "" {
+		return trace.BadParameter("found %d authorities to export, use --out-prefix to export all", l)
+	}
+
+	if a.exportOutputPrefix != "" {
+		perms := os.FileMode(0644)
+		if a.exportPrivateKeys {
+			perms = 0600
+		}
+
+		for i, authority := range authorities {
+			name := fmt.Sprintf("%s%d.cer", a.exportOutputPrefix, i)
+			if err := os.WriteFile(name, authority.Data, perms); err != nil {
+				return trace.Wrap(err)
+			}
+			fmt.Printf("Wrote %s\n", name)
```

Comment:
nit: may want to skip the "Wrote" or output it once at the beginning and/or on stderr to allow for shell operations

Reply:
Sounds good, done.